CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ The following contributions are accepted:
     <td>❌</td>
     <td>
       <ul>
-        <li>Templates editions are allowed with new features additions (but must remain consistent with current visuals)</li>
+        <li>Template changes are allowed with new features additions (but must remain consistent with current visuals)</li>
```

Comment:
Plural agreement...

Reply:
I think `features` get changed to `feature`, too.